### ISSUE TITLE
Whitelist names from the standard lib?

### ISSUE THREAD

**Issue by aileoia** [MichaelAquilina/flake8-spellcheck#43]
Hi, thank you so much for this plugin, I'm looking forward to have only existing words as names inside the code (and have an automatic way of forbidding abbreviations).

Currently however, it is nearly impossible to properly use it, at it throws errors at many of the standard library or third party names, e.g. `pathlib`, `joinpath`, `asdict` not mentioning names defined in packages such as numpy, opencv, pillow.

Would there be a way to spellcheck only variables defined in the module to check?  Or at least provide a way to whitelist the complete standard library...

Thanks in advance!

**Comment by MichaelAquilina** [comment]
Hi @aileoia ! Those should all be part of [python.txt ](https://github.com/MichaelAquilina/flake8-spellcheck/blob/master/flake8_spellcheck/python.txt) so that they are not reported as spelling errors.

would you like to perhaps open a PR to update the file with what you have found?

**Comment by ojomio** [comment]
I use this snippet to remove the mojority of spellcheck false positives from packages

```bash
cat spellcheck_whitelist_in.txt <(python -c 'import sys,pkgutil; [print(i) for i in list(sys.builtin_module_names) + [i.name for i in pkgutil.iter_modules()]]')> spellcheck_whitelist.txt
```

**Comment by MichaelAquilina** [comment]
@ojomio would you like to try opening a PR that adds those into `python.txt`?

**Comment by ojomio** [comment]
That is not enough, as I discovered. We should also add all types of attributes and classes inside those modules, and that's a lot of work. Perhaps, there exists some ready-made solution15:59, 14 июля 2021 г., Michael Aquilina ***@***.***>:
@ojomio would you like to try opening a PR that adds those into python.txt?

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты
